package.json

```diff
@@ -52,6 +52,8 @@
     }
   },
   "dependencies": {
-    "ws": "^7.1.2"
+    "facejs": "^1.0.1",
```

Comment:
This file (and yarn.lock) shouldn't be changed.

Reply:
I think my IDE did that automatically. I just barely even know what those files even do. I'll keep them protected going forward.